src/dmd/backend/cod1.d

```diff
@@ -4678,7 +4685,7 @@ void pushParams(ref CodeBuilder cdb, elem* e, uint stackalign, tym_t tyf)
 
         default:
             break;
-   }
+    }
 
```

Comment:
Eh?

Reply:
Oh thats aligned now. nvm

Reply:
A formatting error that creeped in on another PR. Not worth it's own PR.